recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend

```diff
@@ -30,4 +30,8 @@ do_deploy_append_rcar () {
     if [ -f ${D}/boot/${FLASK_POLICY_FILE} ]; then
         ln -sfr ${DEPLOYDIR}/${FLASK_POLICY_FILE} ${DEPLOYDIR}/xenpolicy
     fi
+
+    if [ -d ${DEPLOYDIR} ]; then
```

Comment:
I believe DEPLOYDIR always exists? Do you need to check?

Reply:
you are right, useless check...